src/dql_lexer.xrl

```diff
@@ -47,8 +39,8 @@ Rules.
 {BUCKET}    :   {token, {kw_bucket,     TokenLine}}.
 {LAST}      :   {token, {kw_last,       TokenLine}}.
 {AS}        :   {token, {kw_as,         TokenLine}}.
-{IN}        :   {token, {kw_in,         TokenLine}}.
```

Comment:
The grafana plugin makes use of the IN keyword in it's queries, is there a new alternative?